docker/service/Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM node:5.6
 # The original repo requires the database to run on the same host as the app
 # This fork/branch allows for configuration of database through environment
 # variables. Thus we can run postgres on a separate host using standard images.
-RUN git clone https://github.com/kasperg/dbc-community-service.git -b env_config /usr/src/app
+# Also we need to update the loopback explorer to get Swagger 2.0 specification.
+RUN git clone https://github.com/kasperg/dbc-community-service.git -b biblo-admin /usr/src/app
```

Comment:
I've read the PR for dbc.-community-service and saw that they have verbally "accepted" the changes. Have they merged them yet? If yes, then this should be changed :relaxed: